### ISSUE TITLE
Unable to download EMNIST

### ISSUE THREAD

**Issue by mugnaidaniele** [Continvvm/continuum#126]
Case 1 :
```python 
from continuum.datasets import EMNIST
dataset_train = EMNIST("data", train=True, download=True)
```
TypeError: __init__() missing 1 required positional argument: 'split'

Case 2 :
```python 
from continuum.datasets import EMNIST
dataset_train = EMNIST("data", train=True, download=True, split="letters")
```
TypeError: __init__() got an unexpected keyword argument 'split'

Case 3 :
```python 
from torchvision.datasets import EMNIST
from continuum.datasets import PyTorchDataset
dataset_train = PyTorchDataset("data", dataset_type=EMNIST, train=True, download=True, split='letters')
```
TypeError: __init__() got an unexpected keyword argument 'split'

**Comment by TLESORT** [comment]
Hi @mugnaidaniele :)
Thanks for the issue we are currently trying to fix the `PyTorchDataset` to solve it. It should be done today, I think.
A potential workaround if your really need it to work now is:
```
from torchvision.datasets import EMNIST
 from continuum.datasets import InMemoryDataset
 pdts = EMNIST(".", train=True, split="letters", download=True) 
dataset_train = InMemoryDataset(pdts.data, pdts.targets) 
```

But the  `PyTorchDataset` should be fix very soon!
Timothée

**Comment by TLESORT** [comment]
We fixed the `PytorchDataset` class in the `continuumDataset` branch, @arthurdouillard will merge it very soon and add it in the pip package :) (thanks Arthur :) )

**Comment by arthurdouillard** [comment]
Hey @mugnaidaniele, thanks for the report. It wasn't a bug per-se, but we didn't add the possibility to pass optionals arguments to Pytorch datasets except the default ones (eg download, path).

@TLESORT just adds this possibility, and it's indeed better. Thank you for your report!

You can update your continuum version to 1.0.15 with `pip install continuum --upgrade`

**Comment by mugnaidaniele** [comment]
I think there is stil a bug:
```python
from torchvision.datasets import EMNIST
from continuum import ClassIncremental
from continuum.datasets import PyTorchDataset

dataset_train = PyTorchDataset("data", dataset_type=EMNIST, train=True, download=True, split='letters')
scenario_train = ClassIncremental(dataset_train, increment=1, initial_increment=1)
``` 
outputs = ufunc(*inputs)
ValueError: 26 is not in list

I think the error is in EMNIST targets; they are 1-based

**Comment by arthurdouillard** [comment]
Wow, that's really weird that PyTorch provides dataset with 1-based indexing... I didn't expect that.
Thank you for your report, I've fixed it in #129 and we'll update pypi (pip) asap

**Comment by arthurdouillard** [comment]
@mugnaidaniele It's fixed, thank for the report.

You can update your continuum version to 1.0.17 with pip install continuum --upgrade
